src/calls/list-trips.ts

```diff
@@ -52,14 +55,22 @@ const tripSchema = z.object({
   notes: z.string().optional(),
   duration: z.number().optional(),
   source: z.enum(['Automatic', 'Manual']),
-});
+}).transform(({commercial_class ,...data}) => ({
+  ...data,
+  commercialClass: commercial_class,
+
+}))
```

Comment:
Here the `vehicle` field, which is an object, has a field in snake case, so that object must also be transformed.